scripts/wmata_data/distcalc.py

```diff
@@ -0,0 +1,149 @@
+        if walkDist <= radiusinmeters:
+            distCsvWriter.writerow((project['Nlihc_id'],'rail',station['Code'],"{0:.2f}".format(getMiles(walkDist))))
+
+
+###### Finds all bus stations within {radiusinmeters} from the given project. Writes to given CSV file.
+def findBusStations(project, radiusinmeters, distCsvWriter):
+    lat = project['Proj_lat']
+    lon = project['Proj_lon']
+
+    params = {'Lat': lat,
+              'Lon' : lon,
+              'Radius': str(radiusinmeters)}
+    response = requests.get('https://api.wmata.com/Bus.svc/json/jStops',params=params, headers=WMATA_HEADERS)
+    data = response.json()
+
+    for stop in data['Stops']:
+        walkDist = getWalkingDistance(lat, lon, str(stop['Lat']), str(stop['Lon']))
+
+        if walkDist <= radiusinmeters: #within 0.5 miles walking
+            distCsvWriter.writerow((project['Nlihc_id'],'bus',stop['StopID'],"{0:.2f}".format(getMiles(walkDist))))
+
+
+####### Writes rail station metadata to given CSV writer
+def writeRailInfo(railStations, infoCsvWriter):
+
+    print("Writing RAIL INFO");
+
+    for station in railStations:
+        #delimit list of lines with colon
+        lines = station['LineCode1'] #all stations have at least one line
+
+        #there is probably a more elegant way to write this...
+        if station['LineCode2'] != None:
+            lines = '{}:{}'.format(lines, station['LineCode2'])
+
+        if station['LineCode3'] != None:
+            lines = '{}:{}'.format(lines, station['LineCode3'])
+
+        if station['LineCode4'] != None:
+            lines = '{}:{}'.format(lines, station['LineCode4'])
+
+        infoCsvWriter.writerow((station['Code'],station['Name'],str(station['Lat']),
+                                                        str(station['Lon']),lines))
+####### Writes bus station metadata to given CSV writer
+def writeBusInfo(infoCsvWriter):
+    print("Writing BUS INFO")
+
+    response = requests.get('https://api.wmata.com/Bus.svc/json/jStops', headers=WMATA_HEADERS)
+    data = response.json()
+
+    for stop in data['Stops']:
+
+        lines = ""
+        for route in stop['Routes']:
+            lines = '{}:{}'.format(lines,route)
+        lines = lines[1:] #take off the first :
+
+        infoCsvWriter.writerow((stop['StopID'],stop['Name'],stop['Lat'],stop['Lon'],lines))
+
+
+
+########### MAIN #############
+def main(csvInputFileName,distOutputFileName,infoOutputFileName):
+
+    projectsFile = open(csvInputFileName)
+    distOutputFile = open(distOutputFileName, 'wt')
+    distCsvWriter = csv.writer(distOutputFile)
+
+    infoOutputFile = open(infoOutputFileName, 'wt')
+    infoCsvWriter = csv.writer(infoOutputFile)
+
+    reader = csv.DictReader(projectsFile)
+
+    numrow = 0
+
+    #print headers
+    distCsvWriter.writerow(('Nlihc_id','type','stop_id_or_station_code','dist_in_miles'))
+    infoCsvWriter.writerow(('code_or_id','type','name','lat','lon','lines'))
+
```

Comment:
It looks like the infoCsvWriter does not have the same data as it's headers indicate. Maybe you refactored to remove 'type' from the data list? if you open wmatainfo.csv 'name' column has 'lat' in it.